src/edit/EditHandle.js

```diff
@@ -35,9 +35,11 @@ L.EditHandle = L.Marker.extend({
 
 	_onHandleDragStart: function() {
 		this._handled.fire('editstart');
+		// this._handled.editing._hideToolbar();
 	},
 
 	_onHandleDragEnd: function() {
+		// this._handled.editing._showToolbar();
```

Comment:
>Just a question, why is this commented out? This shouldn't have any side effects, I guess. Did this introduce bugs?

Same here.

Reply:
Lets move this conversation to the first comment you made on `onHandleDragStart` 👍